opal/mca/btl/tcp/btl_tcp_endpoint.c

```diff
@@ -1044,6 +1044,22 @@ static void mca_btl_tcp_endpoint_recv_handler(int sd, short flags, void *user)
                        .cbdata = reg->cbdata};
                 reg->cbfunc(&frag->btl->super, &desc);
             }
+            else if(MCA_BTL_TCP_HDR_TYPE_PUT == frag->hdr.type && (MCA_BTL_TCP_TAG_PUT_RESP == frag->hdr.base.tag)) {
+                mca_btl_active_message_callback_t *reg = mca_btl_base_active_message_trigger
+                                                         + frag->hdr.base.tag;
+                const mca_btl_base_receive_descriptor_t desc
+                    = {.endpoint = btl_endpoint,
+                       .cbdata = frag->hdr.myself_on_origin};
```

Comment:
That's a bit weird formatting:

```suggestion
                const mca_btl_base_receive_descriptor_t desc = {
                    .endpoint = btl_endpoint,
                    .cbdata = frag->hdr.myself_on_origin
                };
```

Reply:
Thanks! Done and squashed.